components/TaskCard.vue

```diff
@@ -11,13 +11,14 @@
       <div>
         <EditButton />
       </div>
-      <div>
+      <div @click="deleteTask()">
```

Comment:
Please remove the function invocation as you are immediately calling the function.
```suggestion
      <div @click="deleteTask">
```

Reply:
Resolved